training/ansible-testing/e2e-tests/playbook.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Test Environment Provisioning
+  hosts: test_environments
+  remote_user: root
+  become: true
+  gather_facts: false
+
+  tasks:
+  
+  - name: Wait until the instance is ready
+    ansible.builtin.wait_for_connection:
+    delay: 15
+    timeout: 180
+  
+  - name: Gather facts for first time
+    ansible.builtin.setup:
+
+  - name: Clone repository
+    git:
+      repo: https://github.com/instructlab/instructlab.git
+      dest: /tmp/instructlab
+      version: main
```

Comment:
Is there a github release we can use?

Reply:
It appears not, [#1016](https://github.com/instructlab/instructlab/pull/1016) does not show up in the latest release: [v0.14.1](https://github.com/instructlab/instructlab/releases/tag/v0.14.1). For stability I checkout the commit where the tests got merged for now.